pkg/options/network/create.go

```diff
@@ -11,7 +11,12 @@ const (
 	short   = "Network aws cloudformation stack"
 	long    = "Network aws cloudformation stack using predefined yaml templates. "
 	example = `  # Create a aws cloudformation stack
-  rosa create network <template-name> --param Param1=Value1 --param Param2=Value2 `
+  rosa create network <template-name> --param Param1=Value1 --param Param2=Value2 ` +
+		"\n\n" + `  # Rosa quick start HCP VPC example` +
+		"\n" + `  rosa create network rosa-quickstart-default-vpc --param Region=us-west-2` +
+		` --param Name=quickstart-stack --param AvailabilityZoneCount=1 --param VpcCidr=10.0.0.0/16` +
+		"\n\n" + `  # To delete the aws cloudformation stack` +
```

Comment:
NP: `AWS Cloudformation` - we use capitals with acronyms for `HPC VPC` - just for consistency

Reply:
Since we're saying `the ...` rather than referencing the command here